hail/python/hail/experimental/vcf_combiner/vcf_combiner.py

```diff
@@ -486,12 +487,18 @@ def run_combiner(sample_paths: List[str],
         Overwrite output file, if it exists.
     reference_genome : :obj:`str`
         Reference genome for GVCF import.
+    contig_recoding: :obj:`dict` of (:obj:`str`, :obj:`str`)
```

Comment:
or None

Reply:
I used `, optional`, since that seems to be what we do elsewhere.  Let me know if that's no longer preferred.

Or does optional just mean it has a default value, and I should say `dict of ... or None, optional`?

I don't think we're very consistent about this stuff.  :-/

Reply:
I'm not entirely sure what the right practices are. I'll do a bit of research...

And yeah, we really aren't consistent.